### PR TITLE
Allow potting crimson plants

### PR DESCRIPTION
**Pull request by haykam821**
This pull request adds potted versions of crimson saplings, tendrils, and thorns.

### CODE REVIEW

File: src/main/java/modfest/teamgreen/content/block/CrimsonBlocks.java [hunk 1]
```diff
@@ -41,7 +45,11 @@ private CrimsonBlocks(String id, Function<Block.Settings, Block> constructor, Fa
 	private CrimsonBlocks(String id, Function<Block.Settings, Block> constructor, FabricBlockSettings settings, Item.Settings itemSettings) {
 		Identifier identifier = CrimsonInit.from(id);
 		this.block = Registry.register(Registry.BLOCK, identifier, constructor.apply(settings.build()));
-		this.item = Registry.register(Registry.ITEM, identifier, new BlockItem(this.block, itemSettings));
+		if (itemSettings == null) {
```
Comment (valoeghese) [inline src/main/java/modfest/teamgreen/content/block/CrimsonBlocks.java:49]: new line before if statement

Comment (valoeghese) [reply src/main/java/modfest/teamgreen/content/block/CrimsonBlocks.java:49]: also why null item thocc

File: src/main/resources/assets/crimson/lang/en_us.json [hunk 2]
```diff
@@ -21,10 +21,13 @@
   "block.crimson.realgar_ore": "Realgar Ore",
   "block.crimson.vanadinite_ore": "Vanadinite Ore",
   "block.crimson.crimson_thorn": "Crimson Thorn",
+	"block.crimson.potted_crimson_thorn": "Potted Crimson Thorn",
```
Comment (valoeghese) [inline src/main/resources/assets/crimson/lang/en_us.json]: two spaces in json files

File: src/main/resources/assets/crimson/lang/en_us.json [hunk 3]
```diff
@@ -21,10 +21,13 @@
...
   "block.crimson.crimson_log": "Crimson Log",
   "block.crimson.crimson_leaves": "Crimson Leaves",
-  "block.crimson.crimson_sapling": "Crimson Sapling",
-  "block.crimson.crimson_tendrils": "Crimson Tendrils",
+	"block.crimson.crimson_sapling": "Crimson Sapling",
```
Comment (valoeghese) [inline src/main/resources/assets/crimson/lang/en_us.json]: same as above

File: src/main/resources/assets/crimson/models/block/potted_crimson_thorn.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+	"parent": "block/flower_pot_cross",
```
Comment (valoeghese) [inline src/main/resources/assets/crimson/models/block/potted_crimson_thorn.json]: again here

File: src/main/resources/data/minecraft/tags/blocks/flower_pots.json [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{
+	"values": [
+		"crimson:potted_crimson_sapling",
```
Comment (valoeghese) [inline src/main/resources/data/minecraft/tags/blocks/flower_pots.json]: again here

Comment (valoeghese) [reply src/main/resources/data/minecraft/tags/blocks/flower_pots.json]: also don't you need replace=false or something

**Review by valoeghese — CHANGES_REQUESTED** [review]

File: src/main/java/modfest/teamgreen/content/block/CrimsonBlocks.java (hunk 1 above)
Comment (haykam821) [reply src/main/java/modfest/teamgreen/content/block/CrimsonBlocks.java:49]: Flower pots don't have an item.

Comment (haykam821) [reply src/main/java/modfest/teamgreen/content/block/CrimsonBlocks.java:49]: Added the missing new line in 9f64c8574f19b61a662f9ecfd49a927533642067.

File: src/main/resources/assets/crimson/lang/en_us.json (hunk 2 above)
Comment (haykam821) [reply src/main/resources/assets/crimson/lang/en_us.json]: Fixed in 9f64c8574f19b61a662f9ecfd49a927533642067.

File: src/main/resources/assets/crimson/models/block/potted_crimson_thorn.json (hunk 4 above)
Comment (haykam821) [reply src/main/resources/assets/crimson/models/block/potted_crimson_thorn.json]: Fixed in 9f64c8574f19b61a662f9ecfd49a927533642067.

File: src/main/resources/data/minecraft/tags/blocks/flower_pots.json (hunk 5 above)
Comment (haykam821) [reply src/main/resources/data/minecraft/tags/blocks/flower_pots.json]: Fixed in 9f64c8574f19b61a662f9ecfd49a927533642067.

File: src/main/resources/assets/crimson/lang/en_us.json (hunk 3 above)
Comment (haykam821) [reply src/main/resources/assets/crimson/lang/en_us.json]: Fixed in 9f64c8574f19b61a662f9ecfd49a927533642067.

**Review by valoeghese — APPROVED** [review]

**Pull request merged**